data/boarding missions.txt

```diff
@@ -61,6 +61,160 @@ mission "Assisting Merchant (Large)"
 
 
 
+mission "Asteroid Course A"
+	assisting
+	minor
+	to offer
+		random < 10
+	source
+		government "Merchant" "Syndicate"
+		category "Light Freighter" "Interceptor"
+		not attributes "automaton"
+	destination "Earth"
+	on offer
+		conversation
+			`As you are approaching the <origin>, your radar sensors detect that the ship you are assisting is in the middle of a large asteroid's course! While in a normal flight this wouldn't cause much trouble, a ship without active shields and that has already suffered heavy damages could be completely destroyed.`
+			branch unarmed
+				"armament deterrence" == 0
+			choice
+				`	(Shoot towards the asteroid to destroy it.)`
+					goto fire
+				`	(Move your ship between the asteroid and the <origin>, then use your shields to protect it.)`
```

Comment:
Action choices are first person, not second, so this should read "Move my ship..." Maybe also change this to something more along the lines of `(Maneuver my ship to shield the <origin>.)`, as that shortens this up while saying the same thing.